lib/newman_conway.py

```diff
@@ -7,4 +7,16 @@ def newman_conway(num):
         Time Complexity: ?
         Space Complexity: ?
```

Comment:
🤔 Implementation looks good again here, but what about time and space complexity?